dev/travis/script.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Copyright © 2015 Magento. All rights reserved.
+# See COPYING.txt for license details.
+
+set -e
+export PATH="./../../../vendor/bin:$PATH"
```

Comment:
I would argue that it wouldn't really matter much, because in that case, one wouldn't be forced to assume the use of PATH. That said :) I've modified how I did this in 3e8865f. Basically I configured composer to put the vendor binaries in a static location `~/bin` vs `vendor/bin` (which has a different base path depending on the branch name) so that I can set a non-relative location in the PATH env var. Let me know what you think